lib/vsc/administration/vo.py

```diff
@@ -171,37 +174,48 @@ def members(self):
         """Return a list with all the VO members in it."""
         return self.vo.members
 
-    def _get_path(self, storage, mount_point="gpfs"):
+    def _get_path(self, storage, mount_point="backend"):
         """Get the path for the (if any) user directory on the given storage."""
 
         (path, _) = self.storage.path_templates[self.host_institute][storage]['vo'](self.vo.vsc_id)
         if mount_point == "login":
             mount_path = self.storage[self.host_institute][storage].login_mount_point
-        elif mount_point == "gpfs":
-            mount_path = self.storage[self.host_institute][storage].gpfs_mount_point
+        elif mount_point == "backend":
+            mount_path = self.storage[self.host_institute][storage].backend_mount_point
         else:
-            logging.error("mount_point (%s)is not login or gpfs", mount_point)
+            logging.error("mount_point (%s)is not login or backend", mount_point)
             raise Exception()
 
         return os.path.join(mount_path, path)
 
-    def _data_path(self, mount_point="gpfs"):
+    def _data_path(self, mount_point="backend"):
         """Return the path to the VO data fileset on GPFS"""
         return self._get_path(VSC_DATA, mount_point)
 
-    def _data_shared_path(self, mount_point="gpfs"):
+    def _data_shared_path(self, mount_point="backend"):
         """Return the path the VO shared data fileset on GPFS"""
         return self._get_path(VSC_DATA_SHARED, mount_point)
 
-    def _scratch_path(self, storage, mount_point="gpfs"):
+    def _scratch_path(self, storage, mount_point="backend"):
         """Return the path to the VO scratch fileset on GPFS.
 
         @type storage: string
         @param storage: name of the storage we are looking at.
         """
         return self._get_path(storage, mount_point)
 
-    def _create_fileset(self, filesystem_name, path, parent_fileset=None, fileset_name=None, group_owner_id=None):
+    def _get_storage_partition(self, storage_name):
```

Comment:
same as user code?

Reply:
indeed, I made a new parent class for both `VscTier2AccountpageUser` and `VscTier2AccountpageVo` in https://github.com/hpcugent/vsc-administration/pull/134/commits/1f210803bd6c84daff6606f006e96b164bc7903f. It gathers all the code that can be easily shared between them. If you see anything else that can be moved to this parent class let me know.